xbmc/utils/ActorProtocol.h

```diff
@@ -51,7 +51,9 @@ class Message
   bool Reply(int sig, void *data = NULL, int size = 0);
 
 private:
-  Message() {isSync = false; data = NULL; event = NULL; replyMessage = NULL;};
+  Message()
+  : signal(0), isSync(false), isSyncFini(false), isOut(false), isSyncTimeout(false),
```

Comment:
I am not a fan of initializing something for the sake of initializing. before this change the state after construction was obvious, now it isn't anymore. Why should a message defined as not being isOut after construction?